pkg/services/authz/rbac/store/permission_query.sql

```diff
@@ -1,5 +1,11 @@
-SELECT p.action, p.kind, p.attribute, p.identifier, p.scope FROM {{ .Ident .PermissionTable }} as p
-WHERE p.action = {{ .Arg .Query.Action }} AND p.role_id IN (
+SELECT p.kind, p.attribute, p.identifier, p.scope FROM {{ .Ident .PermissionTable }} as p
```

Comment:
Note: I've also removed `action` from the selected attributes - we don't use it anywhere, so there's no need to fetch it.